terra/test/README.md

```diff
@@ -0,0 +1,28 @@
+# Wormhole Contract Test Suite
+
+## Running Local Terra Node
+
+In order to run these tests, you need to have a local Terra node running. These tests are meant to be run using [LocalTerra](https://github.com/terra-money/LocalTerra). This requires [Docker Compose](https://docs.docker.com/compose/install/) to run. You can also run _terrad_ with the same set up Tilt uses (see configuration [here](../../devnet/terra-devnet.yaml)).
+
+## Build
+
+In the [terra root directory](../), run the following:
+```sh
+docker build -f Dockerfile.build -o artifacts .
```

Comment:
could you update these instructions (and the test below too) to use `make`?

Reply:
Good catch. I'll change this